splunk_topology/check.py

```diff
@@ -123,10 +123,10 @@ def check(self, instance):
             if instance.snapshot:
                 self.stop_snapshot(instance_key)
         except Exception as e:
-            self._clear_topology(instance_key, clear_in_snapshot=True)
             self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.CRITICAL, tags=instance.tags, message=str(e))
             self.log.exception("Splunk topology exception: %s" % str(e))
             if not instance.splunk_ignore_saved_search_errors:
```

Comment:
because this is now solved on the lower level, we do not need this here

Reply:
I will keep this as well in case the exception occurs while calling `_saved_searches` api in main check code itself.